book_source/04_advanced_user_guide/02_adding_to_pecan/01_case_studies/02_add_formats.Rmd

```diff
@@ -0,0 +1,127 @@
+# Formats records in BETY {#NewFormat}
+
+The PEcAn database keeps track of all the input files passed to models, as well as any data used in model validation or data assimilation. Before we start to register these files with PEcAn we need to define the format these files will be in.
+
+The main goal is to take all the meta-data we have about a data file and create a record of it that pecan can use as a guide when parsing the data file. 
+
+This information is stored in a Format record in the bety database. Make sure to read through the current Formats before deciding to make a new one. 
+
+# Creating a new Format record in BETY
+
+If the Format you are looking for is not available, you will need to create a new record. Before entering information into the database, you need to be able to answer the following questions about your data:
+
+- What is the file MIME type? We have a suit of functions for loading in data in open formats such as CSV, txt, netCDF, etc. 
+     - PEcAn has partnered with the [NCSA BrownDog project](http://browndog.ncsa.illinois.edu/) to create a service that can read and convert as many data formats as possible. 
+     - If your file type is less common or a proprietary type, you can use the [BrownDog DAP](http://dap.ncsa.illinois.edu/) to convert? 
```

Comment:
This ends with a question mark, but isn't really phrased as a question. It's also not clear whether you're asking the user to convert their data, or to just check whether the data can be converted. Finally, don't forget that Formats loaded by `load_data` are a subset of Formats, since not all data is loaded via load_data. For example, model inputs need to be registered with Formats, even if PEcAn never reads the file, since the Model_Type -> Formats link is what creates the list of optional and required model inputs and allows PEcAn to find existing model input files in in the Inputs table.